ilabs/client/ilabs_datavault_predictor.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, api, datavault, domain, collection, input_facet='input', outp
         self._collection = collection
         self._input_facet=input_facet
         self._output_facet=output_facet
+        self._create_collection_if_needed()
```

Comment:
i do not agree this should be part of the api initialization

user may prefer to receive an error. decision should be made outside the api